### PR TITLE
Rename brainy

### PR DESCRIPTION
**Pull request by optssp**
Hello.

XrostSSP has been renamed brainy.
I changed from XrostSSP to brainy in files and file name.

Please review.

### CODE REVIEW

**Comment by optssp** [comment]
CLA OPT, Inc.

File: extensions/amp-ad/amp-ad.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [AMoAd](../../ads/amoad.md)
 - [AppNexus](../../ads/appnexus.md)
 - [Atomx](../../ads/atomx.md)
+- [brainy](../../ads/brainy.md)
```
Comment (lannka) [inline extensions/amp-ad/amp-ad.md]: is Capital B more appropriate?

File: examples/ads.amp.html [hunk 2]
```diff
@@ -395,6 +395,15 @@ <h2 id="atomx">Atomx</h2>
       data-id="1234">
   </amp-ad>
   
+  <h2 id="brainy">brainy</h2>
+  <amp-ad width="300" height="250"
+      type="brainy"
+      data-aid="10"
+      data-slot-id="3347">
+      <div placeholder></div>
```
Comment (lannka) [inline examples/ads.amp.html]: we indent 4 spaces for line break, otherwise, 2 spaces.

File: extensions/amp-ad/amp-ad.md (hunk 1 above)
Comment (optssp) [inline extensions/amp-ad/amp-ad.md]: our service name is "brainy" and it's all small letters. thanks.

File: examples/ads.amp.html (hunk 2 above)
Comment (optssp) [inline examples/ads.amp.html]: Fixed indent 2 spaces.

**Comment by optssp** [comment]
@lannka Our company's name is OPT, Inc.Could you please check the CLA list again?
Pull #6435

**Comment by lannka** [comment]
@optssp I do see `OPT, inc` on the list.
please help resolve the conflict and we can get it merged.

**Comment by optssp** [comment]
@lannka Thanks to checking the list.
I fixed the conflict.

**Comment by lannka** [comment]
@erwinmombay can you help force merge the PR? CLA robot stalled

**Comment by erwinmombay** [comment]
done.

**Pull request merged**